webpack.common.config.ts

```diff
@@ -653,10 +653,9 @@ module.exports = {
     }]
   },
   optimization: {
+    runtimeChunk: 'single',
```

Comment:
I'm not sure what does the `runtimeChunk: single` do here?
I read about it but I'm not sure what it actually does.

Reply:
It separates the webpack runtime into just this one file, otherwise, the runtime is in all the bundles.

Reply:
Oh wow.
Great.